_posts/events/2016-08-11-mwe.md

```diff
@@ -0,0 +1,74 @@
+
+MWE 2016 will be the 12th event in the series. We will be interested in major challenges in the overall process of MWE treatment, both from the theoretical and the computational viewpoint, focusing on original research related (but not limited) to the following topics:
+ * Lexicon-grammar interface for MWEs
+ * Parsing techniques for MWEs
+ * Hybrid parsing of MWEs
+ * Annotating MWEs in treebanks
+ * Manually and automatically constructed resources
+ * Representation of MWEs in dictionaries and ontologies
+ * MWEs and user interaction
+ * Multilingual acquisition
+ * Multilingualism and MWE processing
+ * Models of first and second language acquisition of MWEs
+ * Crosslinguistic studies on MWEs
+ * The role of MWEs in the domain adaptation of parsers
+ * Integration of MWEs into NLP applications
+ * Evaluation of MWE treatment techniques
+ * Lexical, syntactic or semantic aspects of MWEs
+
+SUMBMISSION MODALITIES
+
+For MWE 2016, we accept the following two types of submissions:
+ * Regular long papers (8 content pages + 2 page for references): Long papers should report on solid and finished research including new experimental results, resources and/or techniques.
+ * Regular short papers (4 content pages + 2 page for references): Short papers should report on small experiments, focused contributions, ongoing research, negative results and/or philosophical discussion.
+
+The reported research should be substantially original. The papers will be presented orally or as posters. The decision as to which papers will be presented orally and which as posters will be made by the program committee based on the nature rather than on the quality of the work. All submissions must be in PDF format and must follow the ACL 2016 formatting requirements (style files available here). We strongly advise the use of the provided Word or LaTeX template files.
+
+Reviewing will be double-blind, and thus no author information should be included in the papers; self-reference should be avoided as well.
+
+Resources submitted with the papers should be anonymized for submission. Papers and/or resources that do not conform to these requirements will be rejected without review. Accepted papers will appear in the workshop proceedings, where no distinction will be made between papers presented orally or as posters.
+
+Submission and reviewing will be electronic, managed by the START system:
+
+https://www.softconf.com/acl2016/mwe
```

Comment:
Put `<` and `>` around the URL so that it is rendered as a link.

Reply:
Done